newIDE/app/src/MainFrame/Providers.js

```diff
@@ -82,52 +83,57 @@ const Providers = ({
-                            >
-                              <PublicProfileProvider>
-                                <I18n update>
-                                  {({ i18n }) => (
-                                    <EventsFunctionsExtensionsProvider
-                                      i18n={i18n}
-                                      makeEventsFunctionCodeWriter={
-                                        makeEventsFunctionCodeWriter
-                                      }
-                                      eventsFunctionsExtensionWriter={
-                                        eventsFunctionsExtensionWriter
-                                      }
-                                      eventsFunctionsExtensionOpener={
-                                        eventsFunctionsExtensionOpener
-                                      }
-                                    >
-                                      <AlertProvider>
-                                        <SubscriptionSuggestionProvider>
-                                          <CommandsContextProvider>
-                                            <AssetStoreStateProvider>
-                                              <ResourceStoreStateProvider>
-                                                <ExampleStoreStateProvider>
-                                                  <ExtensionStoreStateProvider>
-                                                    <TutorialStateProvider>
-                                                      <AnnouncementsFeedStateProvider>
-                                                        <PrivateAssetsAuthorizationProvider>
-                                                          {children({ i18n })}
-                                                        </PrivateAssetsAuthorizationProvider>
-                                                      </AnnouncementsFeedStateProvider>
-                                                    </TutorialStateProvider>
-                                                  </ExtensionStoreStateProvider>
-                                                </ExampleStoreStateProvider>
-                                              </ResourceStoreStateProvider>
-                                            </AssetStoreStateProvider>
-                                          </CommandsContextProvider>
-                                        </SubscriptionSuggestionProvider>
-                                      </AlertProvider>
-                                    </EventsFunctionsExtensionsProvider>
-                                  )}
-                                </I18n>
-                              </PublicProfileProvider>
-                            </AuthenticatedUserProvider>
-                          </InAppTutorialProvider>
+                          <ErrorBoundary
+                            title="GDevelop encountered an issue"
```

Comment:
Can this be translated?

Reply:
I brought the app error boundary above the i18n provider so I would say no 😅

Reply:
oh yes right